apps/prairielearn/src/pages/partials/questionFooter.ejs

```diff
@@ -76,10 +76,20 @@
           <% if (question_context == 'student_exam' && instance_question.points_list) { %>
           <% const rounded_points = instance_question.points_list.map((p) => Math.round(p * 100) / 100); %>
           <small class="font-italic align-self-center text-right">
-              <% if (rounded_points[0] == 1) { %>
-              1 point available for this attempt
+              <% if (instance_question.points == 0) { %> 
```

Comment:
Nitpicking: it is typically good practice in Javascript to [compare with `===` instead of `==` when appropriate](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness). Can you replace this? And since you're working on this file, you may want to replace the following line (which was already using `==`, but could use `===` instead).

Reply:
Thank you for the feedback, made the necessary adjustments